user_modules/shared_testcases/testCasesForExternalUCS.lua

```diff
@@ -0,0 +1,216 @@
+---------------------------------------------------------------------------------------------
+-- Common functions for External_UCS test scripts
+---------------------------------------------------------------------------------------------
+local mobile_session = require("mobile_session")
+local commonPreconditions = require('user_modules/shared_testcases/commonPreconditions')
+local commonFunctions = require('user_modules/shared_testcases/commonFunctions')
+local json = require("modules/json")
+local sdl = require('SDL')
+
+local utils = { }
+
+-- [[ Variables ]]
+
+	utils.HMIAppIds = { }
+	utils.pts = nil
+
+-- [[ Functions ]]
+
+--[[@createTableFromJsonFile: Create table from .json file
+--! @parameters:
+--! file - input file
+--]]
+	function utils.createTableFromJsonFile(file)
+	  local f = io.open(file, "r")
+	  local content = f:read("*all")
+	  f:close()
+	  return json.decode(content)
+	end
+
+--[[@createJsonFileFromTable: Create .json file from table
+--! @parameters:
+--! table - input table
+--! file - output file
+--]]
+	function utils.createJsonFileFromTable(table, file)
+	  local f = io.open(file, "w")
+	  f:write(json.encode(table))
+	  f:close()
+	end
+
+--[[@checkSDLStatus: Checks SDL status against defined one
+--! @parameters:
+--! expStatus - expected status of SDL (0 - STOPPED, 1 - RUNNING)
+--]]
+	function utils.checkSDLStatus(test, expStatus)
+	  local actStatus = sdl:CheckStatusSDL()
+	  print("SDL status: " .. tostring(actStatus))
+	  if actStatus ~= expStatus then
+	    local msg = "Expected SDL status: " .. expStatus .. ", actual: " .. actStatus
+	    test:FailTestCase(msg)
+	  end
+	end
+
+--[[@removeLPT: Delete Local Policy Table
+--! @parameters: NO
+--]]
+	function utils.removeLPT()
+		local folderPath = commonPreconditions:GetPathToSDL() ..
+      commonFunctions:read_parameter_from_smart_device_link_ini("AppStorageFolder")
+    os.execute("rm -rf " .. folderPath)
+	end
+
+--[[@removePTS: Delete Policy Table Snapshot
+--! @parameters: NO
+--]]
+	function utils.removePTS()
+    local filePath = commonFunctions:read_parameter_from_smart_device_link_ini("SystemFilesPath") ..
+      "/" .. commonFunctions:read_parameter_from_smart_device_link_ini("PathToSnapshot")
+    os.execute("rm -rf " .. filePath)
+  end
+
+  local function updatePTU(appId)
```

Comment:
@istoimenova, This is local function (helper) and I think the description is not required since it won't be used outside of the module